### DISCUSSION TITLE
Warning: A valid external Instance reference no longer exists.

### DISCUSSION THREAD

**Discussion by rsc86** [gpuweb/gpuweb#4676] [Q&A] [answered]
Hello i get the warning from the title and the canvas is no longer displayed. Both my browser and gpu-driver are lastest version. How could i narrow down the cause of this behaviouand what could external Instance refer to?

**Comment by Kangz** [comment, marked as answer]
This is a Chromium-specific message and it is not clear without more information might be causing it. If you believe that it is a browser bug, please file an issue on issues.chromium.org with detailed repro steps.

**Reply by rsc86** [reply]
okay thank you! i will take this into consideration. It seems the error is related to gpu lost after around 3800 frames only when working with a sampler and texture_2d in the wgsl shader.
